### PR TITLE
testcase: add testcase 12

### PR DESCRIPTION
**Pull request by wang-mask**
### Ⅰ. Describe what this PR does
testcase: 12 - CSI Plugin Stale Node Patch

### Ⅱ. Does this pull request fix one issue?

add testcase-12 in https://github.com/fluid-cloudnative/community/blob/master/testcase/fluid-testcase.md

### Ⅴ. Special notes for reviews

### CODE REVIEW

File: test/prow/csi_stable_patch.py [hunk 1]
```diff
@@ -0,0 +1,214 @@
...
+    checkAppRun()
+    res = checkLabel(node_name)
+    cleanUp(node_name)
+    print("Has passed? " + str(True))
+    if not res:
+        exit(-1)
+
+
+if __name__ == '__main__':
+    main()
```
Comment (cheyang) [inline test/prow/csi_stable_patch.py:214]: Add return of line.

File: test/prow/csi_stable_patch.py [hunk 2]
```diff
@@ -0,0 +1,215 @@
...
+    api = client.CoreV1Api()
+    resource = api.read_node(node_name)
+    resource.metadata.labels['test-stale'] = None
+    api.patch_node(node_name, resource)
+
+    print("Delete added label.")
+    
+
+def main():
+    config.load_kube_config()
```
Comment (TrafalgarZZZ) [inline test/prow/csi_stable_patch.py:196]: Please change it to `config.load_incluster_config()`

**Review by TrafalgarZZZ — CHANGES_REQUESTED** [review]

**Comment by wang-mask** [comment]
> /test fluid-e2e

what？

**Pull request merged**